resources/js/components/people/gifts/CreateGift.vue

```diff
@@ -109,6 +112,18 @@
           />
         </div>
 
+        <div v-if="displayDate" class="dt dt--fixed pb3 mb3 bb b--gray-monica">
+          <!-- Date -->
+          <form-input
+            :id="'date'"
+            v-model="newGift.date"
+            :input-type="'date'"
+            :class="'dtc pr2'"
+            :title="$t('people.gifts_add_date')"
+            @submit="store"
+          />
```

Comment:
For dates we have a specific component:
```suggestion
          <form-date
            :id="'date'"
            v-model="newGift.date"
            :show-calendar-on-focus="true"
            :locale="locale"
            :class="[ dirltr ? 'fl dtc pr2' : 'fr dtc pr2' ]"
            :label="$t('people.gifts_add_date')"
            @submit="store"
          />
```